dot/node.go

```diff
@@ -106,7 +106,8 @@ func InitNode(cfg *Config) error {
 	r.Stop()
 
 	// TODO: this should be set in the genesis file, not the config
-	if cfg.Core.BabeThreshold == babe.MaxThreshold {
+	// TODO: #1233 confirm
+	if cfg.Core.BabeThresholdNumerator == babe.MaxThreshold {
```

Comment:
you can remove this and directly set the `ThresholdNumerator` and `ThresholdDenominator` in the babe service config

Reply:
which i've seen you've done so you can remove this whole code block